Explain that EIS billing is different to ML models (#3638)
elastic/docs-content: explore-analyze/elastic-inference/eis.md

## Pricing 

All models on EIS incur a charge per million tokens. The pricing details are at our [Pricing page](https://www.elastic.co/pricing/serverless-search) for the Elastic Managed LLM and ELSER.

> Please not that this pricing models differs to the existing [Machine Learning Nodes](https://www.elastic.co/docs/explore-analyze/machine-learning/data-frame-analytics/ml-trained-models) (billed via VCUs consumed).

Reviewer: Typo fix + removing "please" as per [style guide](https://stunning-adventure-qrvr1k2.pages.github.io/style-guide/voice-tone/#please-avoid-please-please)
```suggestion
Note that this pricing models differs from the existing [Machine Learning Nodes](https://www.elastic.co/docs/explore-analyze/machine-learning/data-frame-analytics/ml-trained-models), which is billed via VCUs consumed.
```
